clj-refactor.el

```diff
@@ -3254,6 +3254,17 @@ if REMOVE-PACKAGE_VERSION is t get rid of the (package: 20150828.1048) suffix."
         (replace-regexp-in-string " (.*)" "" version)
       version)))
 
+(defcustom cljr-injected-middleware-version (cljr--version t)
+  "The refactor-nrepl version to be injected.
+
+You can customize this in order to try out new releases.
+
+If customizing it, you most likely should `(setq cljr-suppress-middleware-warnings nil)' too,
+for avoiding a warning that would be irrelevant for this case."
+  :group 'cljr
+  :type 'string
+  :package-version "3.0.0")
```

Comment:
I'm assuming next release will be 3.0.0 btw, for matching the refactor-nrepl version we intend to cut.

Reply:
Yep, I assume so as well. At some point it'd be nice to drop the lockstep in the version, something CIDER did a few years ago. (mostly for the sake of other consumers of the middleware)